solutions/src/main/java/ru/sberbank/school/task08/JacksonSerializableManager.java

```diff
@@ -0,0 +1,76 @@
+package ru.sberbank.school.task08;
+
+import com.esotericsoftware.kryo.io.Input;
+import com.esotericsoftware.kryo.io.Output;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.NonNull;
+import ru.sberbank.school.task08.state.GameObject;
+import ru.sberbank.school.task08.state.InstantiatableEntity;
+import ru.sberbank.school.task08.state.MapState;
+import ru.sberbank.school.task08.state.Savable;
+import ru.sberbank.school.util.Solution;
+
+import java.io.*;
+
+import java.util.List;
+
+@Solution(8)
+public class JacksonSerializableManager extends SaveGameManager {
+
+
+    /**
+     * Класс-наследник должен иметь конструктор эквивалентный этому.
+     *
+     * @param filesDirectoryPath Путь до директории, в которой хранятся файлы сохранений
+     */
+    public JacksonSerializableManager(@NonNull String filesDirectoryPath) {
+        super(filesDirectoryPath);
+    }
+
+
+    @Override
+    public void initialize() {
+
+    }
+
+
+    @Override
+    public void saveGame(String filename, Savable gameState) throws SaveGameException {
+        try (OutputStream outputStream = new FileOutputStream(filesDirectory + filename);
+             Output output = new Output(outputStream)) {
```

Comment:
&#39;try resource&#39; has incorrect indentation level 13, expected level should be 16.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1766898629>*